src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 7.3
 ---
 
+ * Display profiler URL in logs
```

Comment:
We are feature freeze for 7.3, hopefully this feature will be for 7.4. You can add the 7.4 title.